CONTRIBUTING.md

```
@@ -149,6 +146,75 @@ Before starting the release process, ensure you have:
+Any releases must be accompanied by a PR to bump the library install version in the setup/install guide, at minimum.
+
+## Validate website docs snippet changes
+
+To validate that the web documentation code snippets are accurate and up-to-date with the SDK source code, run the following prompt against a locally cloned copy of the [ably/docs](https://github.com/ably/docs) repository and this SDK repository.
+
+> [!IMPORTANT]
+> This prompt should be run with the most powerful LLM available to you (e.g. Claude Opus, GPT-4, etc.) for the best results.
+
+Replace `{DOCS_PATH}` with the path to your local clone of the [ably/docs](https://github.com/ably/docs) repository, `{SDK_NAME}` with `ably-chat-kotlin`, and `{SDK_PATH}` with the path to your local clone of this SDK repository.
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)